src/Aquifer.API/Endpoints/Resources/Content/Helpers.cs

```diff
@@ -68,20 +73,36 @@ public static async Task CreateNewDraft(AquiferDbContext dbContext,
         {
             await historyService.AddAssignedUserHistoryAsync(newResourceContentVersion, assignedUserId, user.Id, ct);
         }
-        else
-        {
-            await historyService.AddSnapshotHistoryAsync(newResourceContentVersion, user.Id, ResourceContentStatus.New, ct);
-        }
 
         var resourceContent = await dbContext.ResourceContents.FindAsync([contentId], ct) ?? throw new ArgumentNullException();
 
-        resourceContent.Status = ResourceContentStatus.AquiferizeEditorReview;
-        if (resourceContentIsUpdating)
+        if (assignedUserId is null || resourceContent.LanguageId == Constants.EnglishLanguageId)
         {
-            resourceContent.Updated = DateTime.UtcNow;
+            await historyService.AddSnapshotHistoryAsync(newResourceContentVersion, user.Id, ResourceContentStatus.New, ct);
         }
 
-        await historyService.AddStatusHistoryAsync(newResourceContentVersion, ResourceContentStatus.AquiferizeEditorReview, user.Id, ct);
+        if (resourceContent.LanguageId == Constants.EnglishLanguageId)
+        {
+            resourceContent.Status = ResourceContentStatus.AquiferizeAwaitingAiDraft;
+            await historyService.AddStatusHistoryAsync(newResourceContentVersion, ResourceContentStatus.AquiferizeAwaitingAiDraft, user.Id, ct);
+            await dbContext.SaveChangesAsync(ct);
```

Comment:
Do we need this? It's been left to the caller otherwise, and where you're calling this calls `SaveChangesAsync` after.

Reply:
Yeah I don't love this either, but since we publish the message right after this I think we need to make sure it's saved so it will always be in the right state when the job gets it. An alternative would be to move the publish message to each of the two usages after the save happens. Thoughts?